pkgs/applications/misc/obsidian-export/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, rustPlatform, fetchgit }:
+rustPlatform.buildRustPackage rec {
+  pname = "obsidian-export";
+  version = "22.11.0";
+
+  src = fetchFromGitHub {
+    owner = "zoni";
+    repo = "obsidian-export";
+    rev = "v${version}";
+    hash =
+      if lib.stdenv.isDarwin
+      then "sha256-mOr8QjuCkiw4rLQH3y/hlo+pPSZVvGyfb5vEGxW2/4g="
+      else "sha256-hfYpSYDA5SsMyk1egvSzKS7/OJ176Z3NHbIT06QqSRc=";
```

Comment:
why the different hash for darwin, out of interest? it's the same repository being downloaded, surely?

Reply:
Yeah, I found that odd, but OfBorg reported a hash mismatch for linux builds. https://logs.nix.ci/?attempt_id=e39149eb-b98e-4a9b-bc25-1590498d9d29&key=nixos%2Fnixpkgs.219334.

After at the logs again today I noticed that for Darwin the file mode for one docs/CONTRIBUTING.md is changed https://logs.nix.ci/?attempt_id=bd42b712-86b9-4508-8693-164a579ec453&key=nixos%2Fnixpkgs.219334

```
 * tag               v22.11.0   -> FETCH_HEAD
T       docs/CONTRIBUTING.md
```

I'd like to avoid the dual hash case so I'll dig around and see if I can address this.